README.md

```diff
@@ -1,71 +1,46 @@
-# Getting Started with Create React App
+# Math Magicians
+In this project I created a simple calculator using React framework.
```

Comment:
- [ ] **OPTIONAL**
It'd be awesome if you attached an image of your app to the README file, it makes it look better and more professional 😄 🙏